publish.sbt

```
@@ -52,6 +51,18 @@ ThisBuild / developers := List(
     email = "miroslav.pojer@absa.africa",
     url = url("https://github.com/miroslavpojer")
   ),
+  Developer(
+    id    = "ABLL526",
+    name  = "Liam Leibrandt",
+    email = "liam.leibrandt@absa.africa",
+    url   = url("https://github.com/ABLL526")
+  ),
+  Developer(
+    id    = "MatloaItumeleng",
+    name  = "Itumeleng Matloa",
+    email = "itumeleng.matloa@absa.africa",
+    url   = url("https://github.com/MatloaItumeleng")
+  ),
```

```suggestion
  Developer(
    id = "jakipatryk",
    name = "Bartlomiej Baj",
    email = "bartlomiej.baj@absa.africa",
    url = url("https://github.com/jakipatryk")
  ),
```
Bart is a contributor, Itu, not yet.